bootstrap/README.md

```diff
@@ -1,42 +1,49 @@
 # Bootstrapping cabal-install
 
-This utility is only intended for use in building cabal-install
-on a new platform. If you already have a functional (if dated) cabal-install
-please rather run `cabal v2-install`.
+This utility is only intended for use in building `cabal-install`
+on a new platform. If you already have a functional cabal-install, however old, then
+please instead run `cabal install cabal-install`.
 
-The typical usage is porting to a new linux architecture,
-then the `linux-{ghc-ver}.json` file is available in the `bootstrap/` folder:
+## Bootstrapping on Linux
 
-On a (linux) system you are bootstrapping, run
+The typical use case is porting to a new Linux architecture. In that case,
+a `linux-${GHCVER}.json` file is available in the `bootstrap/` folder:
 
-   ./bootstrap/bootstrap.py -d ./bootstrap/linux-ghcver.json -w /path/to-ghc
+On a Linux system you are bootstrapping, run
+
+    ./bootstrap/bootstrap.py -d ./bootstrap/linux-${GHCVER}.json -w /path/to-ghc
 
 from the top directory of the source checkout.
 
+### Offline build
+
 For offline builds, you can first run
 
-   ./bootstrap/bootstrap.py -d ./bootstrap/linux-ghcver.json -w /path/to-ghc fetch
+    ./bootstrap/bootstrap.py -d ./bootstrap/linux-${GHCVER}.json -w /path/to-ghc fetch
```

Comment:
Is this a real env variable?

Reply:
If not, maybe delete `$` (I am not sure what the proper style is).

Reply:
It's a question of style and personal preference. I prefer to use env var syntax so that is clear that is not an actual part of the name. Before this PR half of the text was using this style and the other half said just `ghcver`. Again, I think that the former is more clear.

Reply:
For what it's worth, I have the same habit… but it's a Unix thing.

Reply:
Ideally it'd be `_ghc-ver_` but I don't think that combination of markup works

Reply:
Clear enough for me!